app/mailers/rent_mailer.rb

```diff
@@ -1,35 +1,44 @@
 # encoding: UTF-8
 class RentMailer < ActionMailer::Base
-  default from: 'Bilförman <bil@fsektionen.se>', parts_order: ["text/plain", "text/html"]
-  default subject: 'Bilbokning'
-  
-  def rent_email(rent)   
-     @rent = rent     
-     if(@rent.present?) && (@rent.email.present?)
-      mail to: @rent.p_email,
-      subject: %(Bilbokning #{@rent.p_time} (fsektionen.se)),
-      sent_on: Time.zone.now
-    end     
+  default from: %(#{t('rent.foreman')} <bil@fsektionen.se>), parts_order: ["text/plain", "text/html"]
```

Comment:
Line is too long. [101/100]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.